src/agent/debuglet.ts

```diff
@@ -299,6 +299,20 @@ export class Debuglet extends EventEmitter {
       return;
     }
 
+    const workingDir = that.config.workingDirectory;
+    // Don't continue if the working directory is a root directory
+    // unless the user wants to force using the root directory
+    if (!that.config.allowRootAsWorkingDirectory &&
+        path.join(workingDir, '..') === workingDir) {
+      const message =
+          `Refusing to start the cloud debugger with \`workingDirectory\` set to a root ` +
```

Comment:
nit: suggest text: `The working directory is a root directory. Disabling to avoid a scan of the entire filesystem for JavaScript files. Use config \`allowRootAsWorkingDirectory\` if you really want to do this.`